elasticdl/python/worker/task_data_service.py

```diff
@@ -100,19 +100,22 @@ def report_record_done(self, count, err_msg=""):
                 if self._pending_tasks:
                     self._current_task = self._pending_tasks[0]
 
-    def get_validation_dataset(self, eval_task):
+    def get_validation_dataset_gen(self, eval_task):
         """
-        If an evaluation task exists, this creates a `tf.data.Dataset`
-        object as well as its corresponding model version and task_id.
-        Otherwise, this returns `None`.
+        If an evaluation task exists, this creates a generator for further
+        creating `tf.data.Dataset`
```

Comment:
What does "further creating `tf.data.Dataset`" mean?

Reply:
The generator could be used to create a dataset instance. I will polish the comment.